cmd/oras/internal/display/metadata/table/discover_test.go

```diff
@@ -138,3 +139,60 @@ func TestTableDiscoverHandler_Failure(t *testing.T) {
 		t.Errorf("Expected error <%s> actual error <%s>", expected, err.Error())
 	}
 }
+
+type writeFailure struct {
```

Comment:
Should we add it to test utility? Should be frequently reused for mocking writing errors.

Reply:
Done.